### PR TITLE
add zh tutorial for workload-rebalancer

### PR DESCRIPTION
**Pull request by chaosi-zju**
**What type of PR is this?**

/kind documentation

**What this PR does / why we need it**:

add zh tutorial for workload-rebalancer

**Which issue(s) this PR fixes**:
Fixes part of https://github.com/karmada-io/karmada/issues/4840

**Special notes for your reviewer**:

[quick preview](https://deploy-preview-627--karmada.netlify.app/zh/docs/next/tutorials/workload-rebalancer)

### CODE REVIEW

File: i18n/zh/docusaurus-plugin-content-docs/current/tutorials/workload-rebalancer.md [hunk 1]
```diff
@@ -0,0 +1,242 @@
...
+```shell
+git clone https://github.com/karmada-io/karmada
+cd karmada
+hack/local-up-karmada.sh
+export KUBECONFIG=~/.kube/karmada.config:~/.kube/members.config
+```
+
+> **说明：**
+>
+> 在开始之前，我们应该至少安装三个kubernetes集群，一个用于安装 Karmada 控制平面，另外两个作为成员集群。
```
Comment (windsonsea) [inline i18n/zh/docusaurus-plugin-content-docs/current/tutorials/workload-rebalancer.md:27]:
```suggestion
> 在开始之前，我们应该至少安装三个 kubernetes 集群，一个用于安装 Karmada 控制平面，另外两个作为成员集群。
```

File: i18n/zh/docusaurus-plugin-content-docs/current/tutorials/workload-rebalancer.md [hunk 2]
```diff
@@ -0,0 +1,242 @@
...
+> 为了方便，我们直接使用 [hack/local-up-karmada.sh](https://karmada.io/docs/installation/#install-karmada-for-development-environment) 脚本快速准备上述集群。
+>
+> 执行上述命令后，您将看到Karmada控制面和多个成员集群已安装完成。
```
Comment (windsonsea) [inline i18n/zh/docusaurus-plugin-content-docs/current/tutorials/workload-rebalancer.md:30]:
```suggestion
> 执行上述命令后，您将看到 Karmada 控制平面和多个成员集群已安装完成。
```

File: i18n/zh/docusaurus-plugin-content-docs/current/tutorials/workload-rebalancer.md [hunk 3]
```diff
@@ -0,0 +1,242 @@
...
+一般情况下，工作负载类资源被调度后，会保持调度结果不变，其副本分布不会轻易发生变化。
+现在，假设某些特殊情况下，你想主动触发一次重调度，可以通过使用 WorkloadRebalancer 来实现。
+
+因此，本节将指导你如何使用 WorkloadRebalancer 来触发重调度。
+
+## 前提条件
+
+### Karmada 及多个子集群已安装
+
+#### 步骤一: 运行命令
```
Comment (windsonsea) [inline i18n/zh/docusaurus-plugin-content-docs/current/tutorials/workload-rebalancer.md:16]:
```suggestion
#### 运行安装命令
```

File: i18n/zh/docusaurus-plugin-content-docs/current/tutorials/workload-rebalancer.md [hunk 4]
```diff
@@ -0,0 +1,242 @@
...
+你可以通过下述方式来检查该步骤是否成功：
+
+```bash
+$ karmadactl --karmada-context karmada-apiserver get deploy foo
+NAME   CLUSTER   READY   UP-TO-DATE   AVAILABLE   AGE   ADOPTION
+foo    member1   2/2     2            2           20s   Y
+foo    member2   1/1     1            1           20s   Y
+```
+
+可以看到，2个副本分发到 member1 集群，1个副本分发到 member2 集群。
```
Comment (windsonsea) [inline i18n/zh/docusaurus-plugin-content-docs/current/tutorials/workload-rebalancer.md:111]:
```suggestion
可以看到，2 个副本分发到 member1 集群，1 个副本分发到 member2 集群。
```

File: i18n/zh/docusaurus-plugin-content-docs/current/tutorials/workload-rebalancer.md [hunk 5]
```diff
@@ -0,0 +1,242 @@
...
+
+### 第二步：在 member1 集群添加 `NoExecute` 污点以模拟集群故障
+
+* 运行以下命令将 `NoExecute` 污点添加到 member1 集群：
+
+```bash
```
Comment (windsonsea) [inline i18n/zh/docusaurus-plugin-content-docs/current/tutorials/workload-rebalancer.md:117]: indent 2 spaces from row 117 - 128

File: i18n/zh/docusaurus-plugin-content-docs/current/tutorials/workload-rebalancer.md [hunk 6]
```diff
@@ -0,0 +1,242 @@
...
+$ karmadactl --karmada-context karmada-apiserver get deploy foo
+NAME   CLUSTER   READY   UP-TO-DATE   AVAILABLE   AGE   ADOPTION
+foo    member2   3/3     3            3           57s   Y
+```
+
+* 运行以下命令从 member1 集群中移除上述 `NoExecute` 污点：
+
+```bash
```
Comment (windsonsea) [inline i18n/zh/docusaurus-plugin-content-docs/current/tutorials/workload-rebalancer.md:132]: indent 2 spaces from row 132 - 137

**Comment by chaosi-zju** [comment]
@windsonsea thank you for your advice

**Review by RainbowMango — APPROVED** [review]
/lgtm
/approve

**Pull request merged**